### PR TITLE
added css properties to keep title length to one line.

### LINKED ISSUE #1728: Display one line of text only
Are we able to display one line of text only, don't let it wrap. 
If one line is not enough, trim the text. 
Trim based on the length of the text to fit in the space, instead of based on the number of characters, since length could be vary if based on number of characters.

image

### PR DESCRIPTION
**Pull request by NikBorn**
@robbiejaeger
@ds_storoz

This pull request fixes #1728.  The issue requested that the title be on one line only.  We created a new class that had not been used in other parts of the project, `.user-title.` We added the following three css properties to fix the issue: 

``` 
.user-title {
  overflow: hidden;
  text-overflow: ellipsis;
  white-space: nowrap;
}
```

Before:

screen shot 2017-12-05 at 1 55 11 pm

After:

screen shot 2017-12-05 at 1 53 31 pm

### CODE REVIEW

File: src/app/user/list/user-list.component.scss [hunk 1]
```diff
@@ -44,3 +44,11 @@
   text-transform:uppercase;
 }
 
+.user-title {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
+
```
Comment (brittanydionigi) [inline src/app/user/list/user-list.component.scss]: Why are there so many extra blank lines after this class? Looks like there's only 1 blank line in between css selectors elsewhere in the file. Remove these extra blank lines.

File: src/app/user/list/user-list.component.html [hunk 2]
```diff
@@ -78,9 +78,9 @@ <h4 class="content-title">{{paginationConfig.totalItems | i18nPlural: {'=0': 'No
                         </div>
                         <div class="row">
                           <div class="card-metainfo col m6">
-                            <p>{{user.title}}</p>
+                            <p class="user-title">{{user.title}}</p>
                           </div>
-                          <div class="card-metainfo col m6">
+                          <div class="card-metainfo col m6 user-country">
```
Comment (brittanydionigi) [inline src/app/user/list/user-list.component.html]: Is there a reason you're adding a `user-country` class to this element? Seems irrelevant to your changeset?

**Comment by brittanydionigi** [comment]
Does the format of your commit message match the format of other commits in their github history? (e.g. should the first letter be capitalized? should there be punctuation?

**Comment by brittanydionigi** [comment]
Put the code in your comment in between grave marks so that it gets formatted as code. e.g. : We created a new class called `user-title`

**Comment by robbiegreiner** [comment]
@brittanystoroz applied changes to fix your notes.  Previous commit messages from other collaborators don't follow any convention and lack consistency and any description.
